### ISSUE TITLE
Lab 12 - Azure Front Door

### ISSUE THREAD

**Issue by Benjiiim** [MicrosoftLearning/AZ-204-DevelopingSolutionsforMicrosoftAzure#840]
# Lab 12
Now that both flavors of Azure CDN are being retired, should we expect a new version of the lab targeting Azure Front Door?

**Comment by msftnutta** [comment]
![image](https://github.com/user-attachments/assets/9ee80068-5014-4f6e-aecb-0f0580275e75)

I can still see the options for CDN for both Microsoft and Edgio. @Benjiiim

**Comment by Benjiiim** [comment]
Indeed, but it would be better to teach students about current offerings that are going to last for years and not soon-to-be retired services, no?

**Comment by msftnutta** [comment]
@JeffKoMS  what do you think ?

**Comment by JeffKoMS** [comment]
This will be a hot topic for the next certification review/scrub coming up in the next few months. Need to re-evaluate the surface area of where devs typically interact with caching services, and any coverage of Azure CDN will certainly be shifted to focus solely on FrontDoor at that time.
